### ISSUE TITLE
RuntimeError: No stable configuration

### ISSUE THREAD

**Issue by jerzyk** [kennethreitz/pipenv#860]
when converting my old project to Pipenv I've got an exception:

```
RuntimeError: No stable configuration of concrete packages could be found for the given constraints after 16 rounds of resolving.
This is likely a bug.
```

Unfortunately it did not say which package is problematic (even with `--verbose` option) - adding some hints here may be an help.

But going one dependency by one, I've found that removing line:
```
Pillow = "*"
```
resolves an issue, so I changed it to:
```
pillow = "*"
```
and that resolved an issue.

IMHO there is a problem when earlier package is generating dependency with different letter case (in this example `pillow` instead `Pillow`) - so most probably there is an issue with ignoring case of the packages.

**Comment by techalchemy** [comment]
Is this resolved in the latest release of pipenv (8.2.7)?

**Comment by jerzyk** [comment]
it is working now, I can even install ipython!
